xml/book.operations.xml

```diff
@@ -10,10 +10,12 @@
  <info>
   <abstract>
    <para>
-    This guide provides a list of useful procedures for managing your
-    &product; cloud. The audience is the admin-level operator of the
-    cloud.
+     At the time of the &productname; &productnumber; release, this guide
+     contains information pertaining to the operations,
+     administration, and user functions of &productname;.
+     The audience is the admin-level operator of the cloud.
```

Comment:
One thing I want to draw your attentions to @dmpop and @csymons-suse is this. I changed the intro so that our users are aware that this guide doesn't *just* contain operations content (which is quite an important factor with the upstream stuff we're using). Let me know how you feel on wording etc

Reply:
I like it. Maybe "operations" > "operation". Sounds better when I read it aloud.

Reply:
Yeah that makes sense! Thanks Carl :) Will update.